andrey/NewHW1/NewTask1.js

```diff
@@ -0,0 +1,46 @@
+
+function bookManger() {
+    let bookname = window.prompt ("Please, enter a book name: ");
+    let bookauthor = window.prompt ("Pleas, enter a book author: ");
+    let bookyear = window.prompt ("Please enter a book publication year: ");
+    let pagecount = window.prompt ("Please enter a book pages count: ");
+
+    window.confirm("Would you like to add " + bookname + " to the library ?");
+    if (window.confirm = true){
+        window.alert(bookname + " by "  + bookauthor + " successfully saved to the library!")
+    }
+    else {
+
+        }
+
+    return bookLibarary = {
+        name: bookname,
+        author: bookauthor,
+        publication_year: bookyear,
+        pages_count: pagecount,
+        getBookSize: function() {
+            if (pagecount <= 100) {
+               booksize = "Small Size"

```

Comment:
You must not use **booksize** variable to compare, please use returned object property to compare